wpilibc/athena/src/CANJaguar.cpp

```diff
@@ -238,10 +238,8 @@ uint8_t CANJaguar::getDeviceNumber() const { return m_deviceNumber; }
  * <p>In position Mode, the outputValue is in rotations.
  *
  * @param outputValue The set-point to sent to the motor controller.
- * @param syncGroup   The update group to add this Set() to, pending
- *                    UpdateSyncGroup().  If 0, update immediately.
  */
-void CANJaguar::Set(float outputValue, uint8_t syncGroup) {
+void CANJaguar::Set(float outputValue) {
```

Comment:
Might want to order the functions to minimize the diff size as well (for ease of review)